### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by srujanb** [localsend/localsend#1844]
### Is your feature request related to a problem? Please describe.

I like an airdrop feature where once I share an item from my phone to my mac, I get a notification and/or the downloads folder opens up. This is helpful because I usually want to use the shared item immediately. 

### Describe the solution you'd like

It'd be nice to have an option to see a notification "Item received" or something similar, which, when clicked, opens the folder where that item was received. Even more, the item can be focused, selected for more convenience.

**Comment by johandicap** [comment]
I second this! I also often want to use the transferred file immediately, but since there is no link to the folder where it has been downloaded, I need to open a file browser (e.g. Finder on Mac or Nautilus on Ubuntu) and navigate to the LocalSend download folder. Annoys me every time, so I'm glad I found this feature request, and hope someone will pick up on it.

**Comment by srujanb** [comment]
@johandicap fyi this is implemented partially so if the localsend app is open on your mac app, it'd show a popup to open the file. It's not perfect, I'd prefer a notification or just open the folder immediately but slightly improved than before.

**Comment by ShlomoCode** [comment]
This is AirDrop's behavior:

<img width="751" height="240" alt="Image" src="https://github.com/user-attachments/assets/83796bd0-af39-45f7-add3-7b9fdf13ec72" />

But I'm afraid that third-party apps can't easily display such a notification with 2 large buttons

**Comment by johandicap** [comment]
@ShlomoCode Yes, it's something like that that I was thinking of. But I understand that such a popup with multiple buttons might be challenging to implement across platforms.

For now, I discovered that you can click the little History icon and choose Open Folder, which opens the LocalSend downloads folder - so you at least don't have to navigate to it manually.

**Comment by geolaw** [comment]
+1 on this for linux - Fedora + sway + mako (notification daemon) - would be nice to get a notification that something is incoming

On linux I am trying to do this to toggle LocalSend from my scratchpad to be visible
I think this can be using using a background process which watches `dbus-monitor --session "interface='org.freedesktop.Notifications'"` for certain keys and then triggers actions (e.g. `swaymsg scratchpad show`)
